.circleci/config.yml

```diff
@@ -145,7 +145,17 @@ jobs:
       - *docker-cp
       - *build-job
 
-  rtl8730e/flat_dev:
+  rtl8730e/flat_dev_ddr:
+    machine:
+      image: default
+    working_directory: ~/TizenRT
+    steps:
+      - attach_workspace:
+          at: ~/TizenRT
+      - *docker-cp
+      - *build-job
+
+  rtl8730e/flat_dev_psram:
```

Comment:
As you said, they are Realtek internal dev purposes. Even we run just one loadable configuration due to ci runtime, it's difficult to run all. Please keep 1 configuration for them.